src/Stormpath/Oauth/CreateGrantAuthenticationAttempt.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+
+namespace Stormpath\Oauth;
+
+
+use Stormpath\Resource\AccountStore;
+use Stormpath\Resource\Resource;
+
+class CreateGrantAuthenticationAttempt extends Resource
```

Comment:
@ecrisostomo: just referencing the file in Java https://github.com/stormpath/stormpath-sdk-java/blob/token_management/impl/src/main/java/com/stormpath/sdk/impl/oauth/DefaultCreateGrantAuthenticationAttempt.java and the class names here.  public class DefaultCreateGrantAuthenticationAttempt extends AbstractResource.

I remove Deafult and it extends AbstractResource which is our Resource Class